inst/@sym/private/mat_replace.m

```diff
@@ -28,6 +28,17 @@
 
 function z = mat_replace(A, subs, b)
 
+  if islogical(subs{1})
+    subs{1} = find(subs{1});
+    if isempty(subs{1})
+      z = A;
+      return;
+    end
+    if (~isscalar(b) && ~isvector(b))
+      warning('OctSymPy:subsagn:rhs_shape', ...
+            'B neither vector nor scalar in indexed A(I)=B: unusual, did you intend this?')
```

Comment:
Octave double allows it without any warning. Should I remove the warning then? (from here and from the original file `mat_mask_asgn`)